### ISSUE TITLE
Map showing devices around me and where I've seen them before

### ISSUE THREAD

**Issue by dantecatalfamo** [Semper-Viventem/MetaRadar#141]
**Is your feature request related to a problem? Please describe.**
If I have many devices around me, it is very time-consuming to tap on each device and look at its location history.

**Describe the solution you'd like**
I would like to have a view showing my device's location history, the devices around me, and where I have seen them. Instead of having each ping be a bubble like the current device map, each device could have a line connecting all the ping locations.

**Describe alternatives you've considered**
There is another app called rfparty that does something similar, though it has no way to export data lacks a lot of the features present in BLE Radar

**Comment by Cessna207** [comment]
I'm trying to activate the scanner nothing to start the process I don't see nothing to start the app.

**Comment by Semper-Viventem** [comment]
Good idea. History points now supports both markers and path since the app version `0.27.0-bets`

**Comment by dantecatalfamo** [comment]
@Semper-Viventem Thanks a lot! The path view makes it much easier to see where things have been.

I think i may have worded my issue poorly though. What I meant was, it would be nice to have one map that shows the path of my phone, and **_all_** bluetooth devices.

The path for each bluetooth device could be a different color, and the path of the phone could be extra thick, for example. Maybe there could be a filter to only show devices that were near you for more than 10 minutes, or devices that followed you over X number of kilometers.

The problem I have is that I cannot get any real information by looking at the list of 30 bluetooth devices around me, because I have no idea where I've seen them before without clicking on each of them, and with many devices, it's not practical to actually check them all.
